### DISCUSSION TITLE
Wagtail 2.16 support

### DISCUSSION THREAD

**Discussion by winterleitner** [wagtail/wagtail-localize#517] [Q&A] [answered]
After upgrading to wagtail 2.16 I get the following error when doing a pip install:

ERROR: Cannot install wagtail-localize==1.0.1 and wagtail~=2.16 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested wagtail~=2.16
    wagtail-localize 1.0.1 depends on Wagtail<2.16 and >=2.11

Is wagtail localize really incompatible with Wagtail 2.16 ?

**Comment by zerolab** [comment, marked as answer]
The 1.0.1 release is constrained to Wagtail < 2.16. The `main` branch will happily install. So you can use that. Aiming to make a new release towards the end of the week

**Reply by hpoul** [reply]
still plan to make a new release? :-) Are there any blocking issues? 🤔️ thanks

**Reply by zerolab** [reply]
life and real client testing..
Nonetheless, v1.1rc1 is now out https://github.com/wagtail/wagtail-localize/releases/tag/v1.1rc1
